test/upgrade/prober/config.toml

```diff
@@ -0,0 +1,6 @@
+# logLevel = 5 # DEBUG(5)
+[sender]
+address = 'http://default-broker.{{- .Namespace -}}.svc.cluster.local'
```

Comment:
This will only work for soon to be removed ChannelBasedBroker. Can you file an issue to actually fetch the URL from the Broker.Status instead of hardcoding it?

Reply:
ACK